packages/block-library/src/button/editor.scss

```diff
@@ -20,11 +20,6 @@
 		cursor: text;
 	}
 
-	// Don't let placeholder expand parent width.
-	.block-editor-rich-text {
```

Comment:
This is dead code in `master`, and I don't see any regressions.

Reply:
Judging by the comment and in my testing, it is indeed not needed.